lumicks/pylake/tests/test_fdensemble.py

```diff
@@ -36,6 +112,11 @@ def test_iteration_fd_ensemble():
     for fd_a, fd_b in zip(fds.values(), fd_ensemble.values()):
         assert id(fd_a) == id(fd_b)
 
+    # Test that alignment leads to new objects and that the originals weren't modified
```

Comment:
I guess the loop is the one that tests the originals weren't modified, maybe we could join the loops into a single one with the two assertions.

Reply:
👍